lib/components/Alert.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Modal from "./Modal";
+
+export default function Alert({isOpen, title, message, confirmAction, cancelAction, confirmActionText, cancelActionText}) {
+  return (
+    <Modal
+      isOpen={isOpen}
+      size="small"
+      autoHeight
+      showFooter
+      submitButtonProps={{
+        style: "danger",
+        label: confirmActionText || "Proceed",
+        onClick: confirmAction
+      }}
+      cancelButtonProps={{
+        style: "secondary",
+        label: cancelActionText || "Cancel",
+        onClick: cancelAction
+      }}
+      onClose={cancelAction}
+    >
+      <div className="flex">
+        <div className="flex items-center justify-center flex-shrink-0 w-10 h-10 bg-red-100 rounded-full">
+          <i className="text-red-500 ri-alert-line ri-lg"></i>
```

Comment:
Can you make this a prop? Show the alert icon as default though.

Reply:
Fixed